examples/builder_derive_interop/3_hand_subcommand.rs

```diff
@@ -0,0 +1,66 @@
+use clap::error::{Error, ErrorKind};
+use clap::{ArgMatches, Command, FromArgMatches, Parser, Subcommand};
+
+#[derive(Debug)]
+enum CliSub {
+    Add,
+    Remove,
+}
```

Comment:
I can understand  keeping the example focused solely on subcommands but I'm wondering if users will have a hard time seeing how to have arguments for their subcommands without seeing the two parts glued together.

Reply:
Should the subcommand arguments use builder or derive API?